personal-brand/personal-branding.md

```diff
@@ -1,73 +1,178 @@
 # Developing your brand as a software developer
 
-### Soft Skills for Week 2
-
+## Soft Skills for Week 2
 - What are your personal values, and how does your personal brand reflect that?
 - 3 parts of the soft skills answer:
   - Intro - a super short summary of what you will be talking about - give me the punch line right away
   - Story - only tell details that show what you did!
-  - Take away “What I learned from this”, “Going forward I will always”, “The lesson I learned from this situation”...
-
-### Prerequisites
+  - Take away "What I learned from this", "Going forward I will always", "The lesson I learned from this situation"...
 
+## Prerequisites
 - [GitHub account](https://github.com/)
-- [Twitter (or X) account](https://twitter.com)
-- [LinkedIn account](https://linkedin.com)
+- Professional social media presence on at least one of these platforms:
+  - [LinkedIn account](https://www.linkedin.com/)
```

Comment:
Now that this is in a list of "platforms", "account" is a bit confusing here.

Reply:
Actually, we might move LinkedIn out of this list because the lesson itself (appropriately imo) presents LinkedIn and GitHub as the two non-negotiable platforms then recommends at least one of the others.